lib/runtime/wasmer/imports_test.go

```diff
@@ -232,6 +232,59 @@ func Test_ext_offchain_local_storage_clear_version_1_Local(t *testing.T) {
 	require.Nil(t, val)
 }
 
+func Test_ext_offchain_http_request_start_version_1(t *testing.T) {
```

Comment:
`Test_ext_offchain_http_request_start_version_1` is getting failed. Implement `rtm_ext_offchain_http_request_start_version_1` in hostapi and update hostApi wasm url.